packages/gatsby/package.json

```diff
@@ -20,6 +20,15 @@
     "@babel/types": "^7.15.4",
     "@gatsbyjs/reach-router": "^1.3.6",
     "@gatsbyjs/webpack-hot-middleware": "^2.25.2",
+    "@graphql-codegen/add": "^3.1.1",
+    "@graphql-codegen/core": "^2.5.1",
+    "@graphql-codegen/plugin-helpers": "^2.4.2",
+    "@graphql-codegen/typescript": "^2.4.8",
+    "@graphql-codegen/typescript-operations": "^2.3.5",
+    "@graphql-codegen/typescript-resolvers": "^2.6.1",
+    "@graphql-codegen/visitor-plugin-common": "^2.7.4",
```

Comment:
```suggestion
```
Those don't seem like are used anywhere and also don't look like peer deps of other packages?

Reply:
Yeah, those were unused now, good catch